keylime/cloud_verifier_tornado.py

```diff
@@ -270,7 +270,7 @@ def get(self):
                 return
 
             try:
-                agent = session.query(VerfierMain).filter_by(agent_id=agent_id).one_or_none()
+                agent = session.query(VerfierMain).options(joinedload(VerfierMain.ima_policy)).filter_by(agent_id=agent_id).one_or_none()
```

Comment:
`joinedload()` should only be needed if we are going to reference the new `ima_policy` object that's attached to the `agent` object. If it's not used then it's extra overhead on the DB to do the join and marshall the data. Probably not a big deal when dealing with a single object, but it will definitely be noticeable for the bulk responses if there are a lot of entries.

Reply:
I've checked, and all the `joinedload` statements I added are justified - in that they eventually are used to retrieve the `ima_policy` from the agent.

In the bulk case, `process_get_status` (which does ask for `ima_policy`) is called on each agent within the bulk call, so it doesn't seem like we can avoid it for now (short of leaving out any allowlist/IMA policy metadata from the agent status call).